mxabierto/guia
# Add document to list all institutions who has to start with the process

## TODO

- [x] Add new document to "Manuales y Plantillas" menu

## HOW TO TEST

1. Check for the following option under "Manuales y Plantillas" menu

![screen shot 2015-06-26 at 10 18 53 pm](https://cloud.githubusercontent.com/assets/2633527/8390265/abadc2a8-1c51-11e5-9a63-92c26c9e08a2.png)

2. Click on Download File button and validate that indeed downloads a docx file
